src/ScreenshotManager.vala

```diff
@@ -40,7 +40,28 @@ namespace Gala
 
 		public void flash_area (int x, int y, int width, int height)
 		{
-			warning ("FlashArea not implemented");
+			debug ("Flashing area");
+
+			double[] keyframes = { 0.3f, 0.8f };
+			GLib.Value[] values = { 255U, 0U };
+
+			var transition = new Clutter.KeyframeTransition ("opacity");
+			transition.duration = 200;
+			transition.remove_on_complete = true;
+			transition.progress_mode = Clutter.AnimationMode.LINEAR;
+			transition.set_key_frames (keyframes);
+			transition.set_values (values);
+			transition.set_to_value (0.0f);
+			
+			var flash_actor = new Clutter.Actor ();
```

Comment:
Either reuse this actor next time flash_area is called or make sure it gets removed after the transition has finished.